constants.py

```diff
@@ -927,13 +927,15 @@
 
 kIntakeCurrentLimit = (
     CurrentLimitsConfigs()
-    .with_stator_current_limit(25)
+    .with_stator_current_limit(60)
     .with_stator_current_limit_enable(True)
-    .with_supply_current_limit(25)
+    .with_supply_current_limit(60)
     .with_supply_current_limit_enable(True)
 )
 
-kIntakeRealZero = 0  # MAY NEED CHANGING LATER: "I think we just set the starting config, looked at what the encoder said, and offset it by that much" - Landon
+kIntakeRealZero = (
+    -0.25
+)  # MAY NEED CHANGING LATER: "I think we just set the starting config, looked at what the encoder said, and offset it by that much" - Landon
```

Comment:
dawg there are instructions on how to do this
copy from when we set the swerve encoder positions

Reply:
idk where these instructions are + it works 😴

Reply:
https://github.com/1757WestwoodRobotics/2025-Reefscape/blob/c0f035251e76870fc66f59f396b7db706ff97f29/constants.py#L417

Reply:
we did that

Reply:
oop ight I forgot this was about preventing wraparound issues